SPECS/privacyidea-radius.spec

```diff
@@ -50,31 +52,32 @@ Source2: privacyidea-mods-perl
 mkdir -p %{build_dir}/git
 git clone %{gitsource} %{build_dir}/git
 cd %{build_dir}/git; git checkout v%{version}
-mkdir -p $RPM_BUILD_ROOT/usr/lib/privacyidea
-install %{build_dir}/git/privacyidea_radius.pm $RPM_BUILD_ROOT/usr/lib/privacyidea/
-mkdir -p $RPM_BUILD_ROOT/etc/privacyidea
-install %{build_dir}/git/rlm_perl.ini $RPM_BUILD_ROOT/etc/privacyidea/
-install %{build_dir}/git/dictionary.netknights $RPM_BUILD_ROOT/etc/privacyidea/
-mkdir -p $RPM_BUILD_ROOT/etc/raddb/sites-available/
-mkdir -p $RPM_BUILD_ROOT/etc/raddb/mods-available/
-install %{SOURCE1} $RPM_BUILD_ROOT/etc/raddb/sites-available/privacyidea
-install %{SOURCE2} $RPM_BUILD_ROOT/etc/raddb/mods-available/piperl
+# Install necessary files to BUILD_ROOT
+install -D -m 755 %{build_dir}/git/privacyidea_radius.pm $RPM_BUILD_ROOT/usr/lib/privacyidea/privacyidea_radius.pm
+install -D -m 644 %{build_dir}/git/rlm_perl.ini $RPM_BUILD_ROOT/%{_sysconfdir}/privacyidea/rlm_perl.ini
+install -D -m 644 %{build_dir}/git/dictionary.netknights $RPM_BUILD_ROOT/%{_sysconfdir}/privacyidea/dictionary.netknights
+install -D -m 640 %{SOURCE1} $RPM_BUILD_ROOT/%{_sysconfdir}/raddb/sites-available/privacyidea
+install -D -m 640 %{SOURCE2} $RPM_BUILD_ROOT/%{_sysconfdir}/raddb/mods-available/piperl
+# Activate the piperl RADIUS module
+install -d $RPM_BUILD_ROOT/%{_sysconfdir}/raddb/mods-enabled
+cd $RPM_BUILD_ROOT/etc/raddb/mods-enabled && ln -s ../mods-available/piperl .
 
 %post
```

Comment:
In order to have an installation standard like ubuntu, we can also set the link in sites-available as discussed with Paul, and would therefore no longer need to be set manually. Furthermore we unlink existing links like "eap" in mods-enabled and "default" and "inner-tunnel" from sites-enabled but do not delete them from sites-available and mods-available.

Reply:
don't know how to edit code here, have to learn markdown